compiler/next/include/chpl/uast/Variable.h

```diff
@@ -106,7 +104,7 @@ class Variable final : public VarLikeDecl {
   /**
     Returns the kind of the variable (`var` / `const` / `param` etc).
    */
-  Kind kind() const { return this->kind_; }
+  Kind kind() const { return (Kind)((int) storageKind()); }
```

Comment:
They are different enum types that have the same integer values. `Variable::Kind` is a different type from `IntentList`, but we know that the integer values line up (since we built it that way).

I don't know if the cast to `int` is required here but I added it because we were doing it in other cases. At the very least I'm pretty sure it's harmless.